core/src/main/java/novoda/lib/sqliteprovider/sqlite/ExtendedSQLiteOpenHelper.java

```diff
@@ -89,18 +86,6 @@ public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
         return DBUtils.getProjectionMap(getReadableDatabase(), parent, foreignTables);
     }
 
-    /**
-     * Use {@link #getUniqueConstraints(String)}
-     */
-    @Deprecated
-    @Override
-    public List<String> getUniqueConstrains(String table) {
```

Comment:
I was thinking more that a minor release would do it since this method has been deprecated almost two years ago, nobody should still use it. But if the convention dictates that it calls for a major release, I am also ok with that 😂